eucadeploy/plugins/debugger/debuggerplugin.py

```diff
@@ -6,18 +6,27 @@
 from fabric.operations import run, get, settings
 from fabric.state import env
 from fabric.tasks import execute
+from fabric.network import disconnect_all
 import six
 
 
 @six.add_metaclass(abc.ABCMeta)
 class DebuggerPlugin(object):
-    #Base class for example plugin used in the tutorial.
+    #Base class for Debugger Plugin
 
     def __init__(self, component_deployer):
         self.passed = 0
         self.failed = 0
         self.warnings = 0
         self.message_style = "[{0: <20}] {1}"
+        # Minimmum memory requirement for each cloud component in KB
+        self.min_memory_req = 4000000
```

Comment:
Lets put these defaults in their respective plugin files not the base plugin class. Eventually we will want these to be in the config file. That has not yet quite landed yet.